.changeset/gorgeous-pumas-draw.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-analytics-module-newrelic-browser': minor
+'@backstage/plugin-analytics-module-ga4': minor
+'@backstage/plugin-analytics-module-ga': minor
+---
+
+**BREAKING**: Add support to the new analytics api and use `app` as default extension and plugin ids. It is breaking only because you will start seeing app as the new default id for extensions and plugins events.
```

Comment:
As discussed we're making sure this is not a breaking change by still defaulting to the old value. Not that it in practice should've been breaking anyway, but just in case.

Reply:
Done